utils.py

```diff
@@ -95,6 +101,7 @@ def generate_sequence(
         device=device,
     )
 
+    sequence = [tgt_encoder.vocab[idx] for idx in sequence]
     out_sequence = tgt_encoder.untokenize(sequence)
```

Comment:
... because we were missung this line of code

Reply:
It works okay now, but we have old version on master rn

Reply:
Please don't treat github comments like code comments - I'm not able to follow what the issue you're pointing to is with multiple comments across multiple files. If there's an issue, just comment on the line that is causing the issue and reference other sections of the code if necessary :)

Reply:
Sorry :)